mxcube3/routes/ra.py

```diff
@@ -162,19 +162,19 @@ def request_control_response():
     if not data['giveControl']:
         data["sid"] = session.sid
 
-        # Reset request of observer, since it was denied
-        observers = loginutils.get_observers()
+        socketio.emit("setObserver", new_op, room=new_op["socketio_sid"], namespace='/hwr')
```

Comment:
Actually I have some updates on all the code in ra.py, that I will push later. I will accept this for now, but there are some more issues that Ive solved ...